tests/baselines/reference/moduleResolutionNoTsESM.types

```diff
@@ -17,13 +17,13 @@ export default x;
 
 === tests/cases/compiler/user.ts ===
 import x from "./x.ts";
->x : any
+>x : 0
 
 import y from "./y.tsx";
->y : any
+>y : 0
 
 import z from "./z.d.ts";
->z : any
+>z : number

```

Comment:
It is probably worth noting that while these types  refering to a ts source file in the import do now resolve, the error on the import is still present - it's just added in the checker even on successful resolution now.